docs/source/userguide/stepsize.rst

```diff
@@ -28,6 +28,12 @@ To cap the stepsize after some time, the parameters ``t_stop`` and ``stepsize_st
 
     my_stepsize = F.Stepsize(initial_value=1.2, stepsize_change_ratio=1.5, dt_min=1e-6, t_stop=10, stepsize_stop_max=1.5)
 
+Please note that the parameters ``t_stop`` and ``stepsize_stop_max`` will be deprecated in a future release. To set the maximal value of the stepsize, consider using the ``max_stepsize`` parameter:
+
+.. code-block:: python
+
+    my_stepsize = F.Stepsize(initial_value=1.2, stepsize_change_ratio=1.5, dt_min=1e-6, max_stepsize=lambda t: 1 if t < 1 else 2)
```

Comment:
I would change this to a [warning block](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#warnings). I hope the syntax is correct!

```suggestion
.. warning::
    Please note that the parameters ``t_stop`` and ``stepsize_stop_max`` will be deprecated in a future release. To set the maximal value of the stepsize, consider using the ``max_stepsize`` parameter:

    .. code-block:: python

        my_stepsize = F.Stepsize(initial_value=1.2, stepsize_change_ratio=1.5, dt_min=1e-6, max_stepsize=lambda t: 1 if t < 1 else 2)
```

Reply:
Cool! I'll check